RGPosadas/LitterRally
# TASK-34: Setup GraphQL Codegen

This PR closes #126.
 
**Steps to Test**
1. Run `npm i` to install new packages.
2. Check if all GraphQL features behave the same as before (creating events, getting user events, etc...).
3. You can add new queries and mutations to `queries.gql` and `mutations.gql`.
4. Run `npm run gen-graphql` to generate the `graphql.tsx`.

**Notes**
- Pre-commit will always fail on `graphql.tsx`, I don't know if we want to remove it from pre-commit or just run it every time we have to generate new graphql code.
- Some types will probably have to be removed afterwards. This PR only addresses setting up the codegen, so I tried my best not to change the types that were  already there.
